app/Http/Controllers/Frontend/SimBriefController.php

```diff
@@ -85,9 +83,11 @@ public function generate(Request $request)
         }
 
         if ($flight->flight_type === FlightType::CHARTER_PAX_ONLY) {
-            $pax_weight = 197;
+            $pax_weight = setting('simbrief.charter_pax_weight') ?? 168;
```

Comment:
Do `setting('simbrief.charter_pax_weight', 168);`. Same with next line.

Reply:
Will also do a cleanup in controller while fixing those

Reply:
Completed